README.md

```diff
@@ -1,5 +1,4 @@
-# Light Table [![Build Status](https://travis-ci.org/LightTable/LightTable.svg?branch=master)](https://travis-ci.org/LightTable/LightTable)
-
+# Light Table [![Build Status](https://travis-ci.org/LightTable/LightTable.svg?branch=master)](https://travis-ci.org/LightTable/LightTable) [![CircleCI](https://circleci.com/gh/Dublo/LightTable-forked.svg?style=svg)](https://circleci.com/gh/Dublo/LightTable-forked)
```

Comment:
> Dublo/LightTable-forked

This is wrong. You will have to create an official account. In the meantime it will do.

Reply:
Right. Forgot about it. I'll create a new PR on CI builds and other integrations for now...